include/domain/impl/sources/elastic/elastic2d_anisotropic.tpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "compute/interface.hpp"
+#include "domain/impl/sources/elastic/elastic2d_anisotropic.hpp"
+#include "domain/impl/sources/source.hpp"
+#include "enumerations/interface.hpp"
+#include "globals.h"
+#include "kokkos_abstractions.h"
+#include "source_time_function/source_time_function.hpp"
+#include "specfem_setup.hpp"
+#include <Kokkos_Core.hpp>
+
+// -----------------------------------------------------------------------------
+//                     SPECIALIZED ELEMENT
+// -----------------------------------------------------------------------------
+
+// template <int NGLL>
+// KOKKOS_FUNCTION specfem::domain::impl::sources::source<
+//     specfem::enums::element::dimension::dim2,
+//     specfem::enums::element::medium::elastic,
+//     specfem::enums::element::quadrature::static_quadrature_points<NGLL>,
+//     specfem::enums::element::property::isotropic>::
+//     source(const specfem::compute::properties &properties,
+//            specfem::kokkos::DeviceView4d<type_real> source_array) {
+
+//   // #ifndef NDEBUG
+//   //   assert(source_array.extent(1) == NGLL);
+//   //   assert(source_array.extent(2) == NGLL);
+//   // #endif
+
+//   return;
+// }
+
+template <int NGLL, bool using_simd>
+KOKKOS_INLINE_FUNCTION void specfem::domain::impl::sources::source<
+    specfem::dimension::type::dim2, specfem::element::medium_tag::elastic,
+    specfem::element::property_tag::anisotropic,
+    specfem::enums::element::quadrature::static_quadrature_points<NGLL>,
+    using_simd>::
+    compute_interaction(
+        const specfem::datatype::ScalarPointViewType<
+            type_real, components, using_simd> &stf,
+        const specfem::datatype::ScalarPointViewType<
+            type_real, components, using_simd>
+            &lagrange_interpolant,
+        specfem::datatype::ScalarPointViewType<
+            type_real, components, using_simd> &acceleration)
+        const {
+
+  if constexpr (specfem::globals::simulation_wave == specfem::wave::p_sv) {
```

Comment:
Isn't this identical to the isotropic case? Could you remind me why there is difference in source implementation for isotropic/anisotropic cases?

Reply:
Yes it is identical. This was my bad when I designed this. Sources should only be dependent on medium type and not property. I have another task in refactoring section to address this.

Reply:
Saw that too late! Whoops